styles/components/_select2.scss

```diff
@@ -22,3 +22,72 @@
     }
   }
 }
+
+// Tags (the element that select2 is generated off of must have a class of 'select2Tags').
+// Focus styling is with other form focus styling in form.scss.
+.select2TagsWrap {
+  position: relative;
+
+  .select2Tags + .select2-container {
+    &.select2-container--default {
+      .select2-selection--multiple .select2-selection__rendered {
+        padding-bottom: $padSm;
+        border-radius: 3px;
+      }
```

Comment:
The .select2-selection--multiple element is getting a height of 4em, that makes it shorter than the UL and LI elements, which throws off other measurements. 

If you gave it a height of auto, that would cause it to contain the UL and LI, instead of being shorter.

```
.select2-selection--multiple {
        height: auto;

        .select2-selection__rendered {
          padding-bottom: $padSm;
          border-radius: 3px;
        }
      }
```

The height of the control is jumping up when a tag is added, and then going back down when all tags are removed, this is because the input is shorter than the tags. That could be fixed by giving it a line-height to make it the same height (inside .select2-container--default). 

```
.select2-search--inline .select2-search__field {
        line-height: 42px;
      }
```